ios/Runner/Info.plist

```diff
@@ -41,5 +43,8 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<!-- TODO verify this API usage and explanation -->
```

Comment:
Remove comment. They are not this strict, just explain it right and they will approve.

Reply:
This is more for consistency once we start adding additional API usages.